polkadot/node/core/pvf/src/artifacts.rs

```diff
@@ -169,6 +169,15 @@ pub struct Artifacts {
 	inner: HashMap<ArtifactId, ArtifactState>,
 }
 
+/// A condition which we use to cleanup artifacts
+#[derive(Debug)]
+pub enum CleanupBy {
+	// Inactive time after which artefact is deleted
+	Time(Duration),
+	// Max size in bytes. Reaching it older artefacts are deleted
```

Comment:
The comment seems wrong, we delete least used ones. At least for me `older` usually means in relation to creation time.

Reply:
Fixed